### PR TITLE
Update dependencies (from Angular 11 to Angular 13 via angular/cli update)

### PR DESCRIPTION
**Pull request by Qwby**

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -18,15 +18,20 @@
             "main": "src/main.ts",
             "polyfills": "src/polyfills.ts",
             "tsConfig": "tsconfig.app.json",
-            "aot": true,
```
Comment (gorzala) [inline angular.json:21]: Why did you remove this?

**Review by gorzala — APPROVED** [review]

File: angular.json (hunk 1 above)
Comment (Qwby) [reply angular.json:21]: The automated migration from Angular 11 to 12 removed this attribute, because aot compiler option is true per default.
